api/apps/api/src/modules/published-project/entities/published-project.api.entity.ts

```diff
@@ -5,6 +5,7 @@ import {
   Creator,
   Resource,
 } from '../dto/create-published-project.dto';
+import { string } from 'fp-ts';
```

Comment:
I don't think this is what you meant (though it may be what LSP and vscode thought you meant)